build_projects/update-dependencies/Program.cs

```diff
@@ -1,25 +1,93 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.DotNet.Cli.Build.Framework;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.VersionTools;
+using Microsoft.DotNet.VersionTools.Automation;
+using Microsoft.DotNet.VersionTools.Dependencies;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace Microsoft.DotNet.Scripts
 {
     public class Program
     {
-        public static int Main(string[] args)
+        private static readonly Config s_config = Config.Instance;
+
+        public static void Main(string[] args)
         {
             DebugHelper.HandleDebugSwitch(ref args);
 
-            return BuildSetup.Create(".NET CLI Dependency Updater")
-                .UseTargets(new[]
-                {
-                    new BuildTarget("Default", "Dependency Updater Goals", new [] { "UpdateFiles", "PushPR" }),
-                    new BuildTarget("UpdateFiles", "Dependency Updater Goals"),
-                    new BuildTarget("PushPR", "Dependency Updater Goals"),
-                })
-                .UseAllTargetsFromAssembly<Program>()
-                .Run(args);
+            List<BuildInfo> buildInfos = new List<BuildInfo>();
+
+            buildInfos.Add(BuildInfo.Get("CoreFx", s_config.CoreFxVersionUrl, fetchLatestReleaseFile: false));
```

Comment:
This is a weird piece of code. Just saying. Is the value from s_config the fallback one in case BuildInfo does not have a value for what you are trying to get?